Zend/zend_API.h

```diff
@@ -674,6 +674,7 @@ END_EXTERN_C()
 #define RETURN_ZVAL(zv, copy, dtor)		do { RETVAL_ZVAL(zv, copy, dtor); return; } while (0)
 #define RETURN_FALSE  					do { RETVAL_FALSE; return; } while (0)
 #define RETURN_TRUE   					do { RETVAL_TRUE; return; } while (0)
+#define RETURN_THROWS()					ZEND_ASSERT(EG(exception)); return
```

Comment:
Have you considered to wrap that in `do {...} while (0)`, so the macro is expanded to a single statement? It also might be worth to consider whether to drop the empty parameter list.

Generally, I'm very much in favor of having a macro for this purpose. Thanks!

Reply:
Empty parameter list should be present, RETURN_FALSE/TRUE don't have one because legacy.

Reply:
@cmb69 Yes, I did, but I didn't know why it is useful (I thought that it has to do something with scopes), so skipped the idea. Now, I found https://github.com/php/php-src/pull/3116#pullrequestreview-97655832 so it's clear :)